src/mapInteractor.js

```diff
@@ -636,7 +636,7 @@ var mapInteractor = function (args) {
         (!m_mouse.buttons.left || m_options.click.buttons.left) &&
         (!m_mouse.buttons.right || m_options.click.buttons.right) &&
         (!m_mouse.buttons.middle || m_options.click.buttons.middle)) {
-      m_clickMaybe = true;
+      m_clickMaybe = {x: m_mouse.page.x, y: m_mouse.page.y};
```

Comment:
We can receive a mousemove event from some browsers where there is no movement.  We need to record where the mouse down occurred, but we only care until we decide that a click is impossible.  At present, a click only gets passed through if there is no movement.  We could make it so that there is a small threshold of movement that would not trigger moving events, but would still allow clicks.